contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Understanding page load time metrics in performance monitoring
```

Comment:
Can't put my finger on it but the title isn't quite working for me - it's somehow long yet not descriptive enough?

Something along the lines of 'How to figure out what's slowing your website down using PostHog' or similar might be a bit more of a hook?

Reply:
"in performance monitoring" might not be necessary